### ISSUE TITLE
Laravel 6.0 support if it works, please!

### ISSUE THREAD

**Issue by robert1233** [svenluijten/artisan-view#59]
Would love to continue using this package in 6.0. Thanks for all your work!

**Comment by svenluijten** [comment]
Hey Robert,

I will research & tag a new release as soon as I'm able to, which I expect to be later today! Thanks for your interest in the package 😄

**Comment by NissiusRibas** [comment]
I'm waiting too ;)

**Comment by svenluijten** [comment]
I'm not able to run phpunit on this project because of a dependency blocking the installation of Laravel 6.0. Will have to put this off for a bit. In the meantime, code fixes I pushed to the `3.0` branch did make it compatible already.

To use it _now_ with Laravel 6.0, update the version constraint in `composer.json` to `"sven/artisan-view": "^3.0 || 3.0.x-dev"`, and run `composer update sven/artisan-view`. After I've tagged a new release, you'll be able to remove the `3.0.x-dev` part there.

I'll update this issue once I've tagged the new version 🙂
